### PR TITLE
Update settingUp.md #5131

### LINKED ISSUE #5131: SettingUp.md: recommend Source Tree instead of GitHub for Windows
The current SettingUp.md says `Install GitHub for Windows/Mac (recommended), or at least, Git`

Should be changed as `Install Source Tree or other similar Git Client, or at least Git`.

### PR DESCRIPTION
**Pull request by ZhaoDingfan**
Fixes #5131 

**Outline of Solution**

Basically just change the sentence in settingUp.md, commit and push it.

### CODE REVIEW

**Comment by ZhaoDingfan** [comment]
@xpdavid This is the new pull request. Ready for review.

File: docs/settingUp.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
    ```sh
    git remote add upstream https://github.com/TEAMMATES/teammates.git
    ```
-
```
Comment (xpdavid) [inline docs/settingUp.md]: Why you need to change this?

Comment (ZhaoDingfan) [inline docs/settingUp.md]: This is an auto-generated change, I don't know why it changes. And it seems that the content doesn't change.

Comment (xpdavid) [inline docs/settingUp.md]: It's the whitespace. Usually, it is your editor that makes this kind of changes for you.

Comment (ZhaoDingfan) [inline docs/settingUp.md]: I got the idea. Thanks for your notice.

**Comment by wkurniawan07** [comment]
The issue at hand has been resolved by #6960. @ZhaoDingfan you may wish to work on another `d.FirstTimers` issue.

**Pull request closed without merging**